test/Elastic.Apm.Tests/ConstructorTests.cs

```diff
@@ -155,6 +155,12 @@ private class LogConfiguration : IConfiguration, IConfigurationDescription
 
 		public bool UsePathAsTransactionName => ConfigConsts.DefaultValues.UsePathAsTransactionName;
 
+		public Uri ProxyUrl => ConfigConsts.DefaultValues.ProxyUrl;
```

Comment:
Rather than constants I think we could have a readonly autoproperty for these? e.g.

```suggestion
		public Uri ProxyUrl { get; }
```

Reply:
yes, done